language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionUtil.java

```diff
@@ -307,7 +307,7 @@ public static CodeActionNodeType codeActionNodeType(Node node) {
         }
 
         // Remove brackets of the unions, except the nil "()" type
-        types = types.stream().map(v -> v.replaceAll("^\\((.+)\\)$", "$1")).collect(Collectors.toList());
+//        types = types.stream().map(v -> v.replaceAll("^\\((.+)\\)$", "$1")).collect(Collectors.toList());
```

Comment:
if not needed shall we remove this line instead of commenting?

Reply:
Yes, we can remove this. Done.